mobify/platform-scaffold
# AMP-61: List Component

This PR adds the `List` component that is needed for the _PLP_ in *AMP-61*, it isn't the build-out of the _PLP_ itself.

 **JIRA**: https://mobify.atlassian.net/browse/AMP-61

## Changes
- Added in `List` component, related tests and READMEs
- Added in `amp-list__item` class, a `TODO` in PWA `List` SCSS
- Removed empty SCSS files

## Todos
- [ ] Engi +1

## How to test-drive this PR
- (necessary config changes)
- Run `npm run dev`
- Visit `localhost:3000`
- I added a `List` with both `items` and a `List` with `ListTile` to the template to see if they worked 👍
